contracts/Gap.sol

```diff
@@ -248,4 +180,100 @@ contract Gap is Initializable, OwnableUpgradeable, EIP712Upgradeable {
             totalUids[i] = eas.multiAttest(requests);
         }
     }
+
+    ///
+    /// Perform a single attestation
+    ///
+    function _attest(
+        AttestationRequest calldata request,
+        address attester
+    ) private returns (bytes32) {
+        AttestationRequestData[]
+            memory requestData = new AttestationRequestData[](1);
+        requestData[0] = request.data;
+        validateCanAttestToRefs(requestData, attester);
+
+        return eas.attest(request);
```

Comment:
What is stopping someone from bypassing our method and directly calling this?

Reply:
The resolver we set on the schema